packages/editor/api-report.md

```diff
@@ -2584,6 +2585,7 @@ export interface TLCameraMoveOptions {
 export interface TLCameraOptions {
     constraints?: TLCameraConstraints;
     isLocked: boolean;
+    isZoomLocked: boolean;
```

Comment:
Also, do we want to test that the user panning with trackpad also doesn't do anything?

Reply:
i dont think thats considered zooming no?